packages/spright-components/src/chat/input/tests/chat-input.spec.ts

```diff
@@ -205,53 +210,6 @@ describe('ChatInput', () => {
         });
     });
 
-    describe('resetInput method', () => {
-        beforeEach(async () => {
-            await connect();
-        });
-
-        it('clears the input contents, sets focus, and disables the button', () => {
-            page.setText('new value');
-            element.resetInput();
-            processUpdates();
-
-            expect(element.value).toEqual('');
-            expect(page.getRenderedText()).toEqual('');
-            expect(page.textAreaHasFocus()).toBeTrue();
-            expect(page.isSendButtonEnabled()).toBeFalse();
-        });
-
-        it('can be called from send event handler', async () => {
```

Comment:
Think it would be worth adding an assert to one or more of the `send` event tests to assert the state of `element.value` when the event is fired? Basically asserting that we reset it before firing the event, not after.